### ISSUE TITLE
[MediaQuery] Queries are duplicated all the time a component is initialized.

### ISSUE THREAD

**Issue by philippbro** [zurb/foundation-sites#10736]
#### How to reproduce this bug:

1. Initialize Foundation once with e.g. 3 breakpoints -> results in 3 MediaQueries (small, medium, large)
2. Call `new Foundation.Tooltip($element, {});` once 

#### What should happen:

Only 3 MediaQueries are available and old ones are updated.

#### What happened instead:

There are 6 MediaQueries available (2 x small, 2 x medium, 2 x large)

#### Browser(s) and Device(s) tested on:

Google Chrome Version 61.0.3163.100 (64-Bit)

#### Foundation Version(s) you are using:

6.4.3

**Comment by SassNinja** [comment]
I just came across the same issue while working with the media query util.

Still present at the current develop branch.
You can easily test this on the [docs homepage](https://foundation.zurb.com/sites/docs/) where `Foundation.MediaQuery.queries` should return an array with only 5 items (currently it returns 35 items).

```javascript
let myQueryNames = [];
let myQueryValues = [];
let duplicateCount = 0;

Foundation.MediaQuery.queries.forEach(query => {

    if (myQueryNames.indexOf(query.name) > -1 || myQueryValues.indexOf(query.value) > -1) {
        duplicateCount++;
    }

    if (myQueryNames.indexOf(query.name) > -1) {
        console.log('DUPLICATE NAME',query.name);
    } else {
        myQueryNames.push(query.name);
    }

    if (myQueryValues.indexOf(query.value) > -1) {
        console.log('DUPLICATE VALUE',query.value);
    } else {
        myQueryValues.push(query.value);
    }
});

console.log(`--------\nDuplicates: ${duplicateCount}\n--------`);
console.log(myQueryNames.map((name, index) => `[${name}] ${myQueryValues[index]}`).join('\n'));
```

**Comment by icrismariu** [comment]
Bump! this is still going on as Dec 2018 ... it's this a feature or a bug? :)

**Comment by DanielRuf** [comment]
> Bump! this is still going on as Dec 2018 ... it's this a feature or a bug? :)

Also with 6.5.1?

**Comment by icrismariu** [comment]
> > Bump! this is still going on as Dec 2018 ... it's this a feature or a bug? :)
> 
> Also with 6.5.1?

yep

**Comment by ncoden** [comment]
See https://github.com/zurb/foundation-sites/pull/11543#issuecomment-430941528

What is seen as a bug here can also be considered as a feature: media-queries are kind of refreshed everytime a component is initialized.

Prevent it is could break some websites and developers may need to manually call `MediaQuery.reInit();` to refresh media-queries. So it is planned for v6.6.0.

**Comment by DanielRuf** [comment]
Sure but shouldn't the list be cleaned up of duplicates then?

**Comment by ncoden** [comment]
You can provide a PR if you want for v6.5 but for now nothing is broken.

**Comment by ncoden** [comment]
Duplicate of https://github.com/zurb/foundation-sites/issues/11540 and fixed in https://github.com/zurb/foundation-sites/pull/11543. Will be released in v6.6.0. Closing.
